qcodes/instrument_drivers/tektronix/AWG5014.py

```diff
@@ -903,7 +906,7 @@ def generate_channel_cfg(self):
         been changed from their default value and put them in a
         dictionary that can easily be written into an awg file, so as
         to prevent said awg file from falling back to default values.
-        (See self.generate_awg_file and self.AWG_FILE_FORMAT_CHANNEL)
+        (See self.make_awg_file and self.AWG_FILE_FORMAT_CHANNEL)
```

Comment:
You could use actual sphinx links here. https://kev.inburke.com/kevin/sphinx-interlinks/